times_reader/transforms.py

```diff
@@ -2323,6 +2408,21 @@ def rename_cgs(
     return output
 
 
+def fix_topology(
+    config: datatypes.Config, input: Dict[str, DataFrame]
+) -> Dict[str, DataFrame]:
+    output = {}
+
+    mapping = {"IN-A": "IN", "OUT-A": "OUT"}
+
+    for table_type, df in input.items():
+        if table_type == "TOPOLOGY":
+            df["io"].replace(mapping, inplace=True)
+        output[table_type] = df
+
+    return output
```

Comment:
```suggestion
    mapping = {"IN-A": "IN", "OUT-A": "OUT"}

    if "TOPOLOGY" in input:
        input["TOPOLOGY"]["io"].replace(mapping, inplace=True)

    return input
```
Yes, you don't need to iterate through all the items in a dictionary, it is much faster, O(1), to query if a key is present in a dictionary. You can then update the `input` dictionary directly since dictionaries are mutable. (I notice that many transforms are needlessly creating a new `output` dictionary, I'll open a PR to fix them.)